src/libcore/sync/atomic.rs

```diff
@@ -1931,12 +1972,12 @@ atomic_int! {
 }
 #[cfg(target_has_atomic = "64")]
 atomic_int! {
-    unstable(feature = "integer_atomics", issue = "32976"),
-    unstable(feature = "integer_atomics", issue = "32976"),
-    unstable(feature = "integer_atomics", issue = "32976"),
-    unstable(feature = "integer_atomics", issue = "32976"),
-    unstable(feature = "integer_atomics", issue = "32976"),
-    unstable(feature = "integer_atomics", issue = "32976"),
+    stable(feature = "integer_atomics_stable", since = "1.33.0"),
+    stable(feature = "integer_atomics_stable", since = "1.33.0"),
+    stable(feature = "integer_atomics_stable", since = "1.33.0"),
+    stable(feature = "integer_atomics_stable", since = "1.33.0"),
+    stable(feature = "integer_atomics_stable", since = "1.33.0"),
+    stable(feature = "integer_atomics_stable", since = "1.33.0"),
```

Comment:
If you want to wait out the full FCP duration (10 days) then this will miss the 1.33 master=>beta promotion date which is on the 15th of January (FCP completes around 18-19th).

Reply:
I guess this needs to be changed to 1.34.0 then ;)